data/kahet/aberrant outfits.txt

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Weapons:
+
+
+outfit "Aberrant Spiker"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	"nettle capacity" 60
+	weapon
+		sprite "projectile/aberrant spike"
+			"frame rate" 24
+			"scale" 0.5
+			"rewind"
+		sound "finisher"
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 2.4
+		"reload" 60
+		"firing energy" 6
+		"firing heat" 3
+		"velocity" 6
+		"lifetime" 1800
+		"acceleration" 0.6
+		"drag" .09
+		"turn" 3
+		"homing" 3
+		"optical tracking" 0.8
+		"hit force" 300
+		"hull damage" 1000
+		"shield damage" 1000
+		"missile strength" 12
+	description "Spikes grow on the backs of many Aberrant variants, made of a seemingly random assortment of materials both metallic and mineral in nature. This device appears to allow an Aberrant to launch these spikes as an offensive weapon, but just how they are able to guide and propel themselves through space is a mystery that a cursory examination of this outfit does not reveal."
+
+
+outfit "Nucleolysis Beam"
+	category "Guns"
+	cost 1800000
+	thumbnail "outfit/unknown"
+	"flotsam sprite" "outfit/unknown"
+	"mass" 29
+	"outfit space" -23
+	"weapon capacity" -23
+	"flotsam chance" 0.5
+	"unplunderable" 1
+	"gun ports" -1
+	weapon
+		sprite "projectile/nucleolysis beam"
+			"frame rate" 15
+		sound "fire-lance"
+		"hit effect" "acuit hit"
+		"inaccuracy" .2
+		"velocity" 380
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 4.2
+		"firing heat" 1.5
+		"shield damage" 0.8
+		"hull damage" 1.6
+		"heat damage" -9.1
+		"fuel damage" 0.9
+		"corrosion damage" 0.15
+	description "This beam weapon creates a field inside which otherwise stable atomic nuclei rapidly decay. This effect seems to be inversely proportional to the mass of the nucleus, causing isotopes like deuterium to break down at a higher rate than heavier elements. As the resultant products have a higher energy state than the original nucleus, the beam cools its target, defying conventional thermodynamics."
+	description "In practice, this beam can deplete a ship's deuterium fuel while causing the very matter of its hull to decay."
+
+
+outfit "Telescopic Antimissile"
```

Comment:
Fires more slowly and with weaker shots than the PDT, sits between the Wanderer AM and the Warder in terms of the tradeoff between AM strength and rate of fire with the same 120 AM/sec as both.
The extremely long range is a strength, and makes this AM very difficult to defeat by having a single missile cross its full range without being shot down, but it's no help against being overwhelmed by missiles fired faster than the AM can shoot them down, or against missiles fired from close-mid ranges.
Overall, I think this is a well-balanced outfit.